packages/rum-core/src/domain/profiling/lazyLoadProfiler.ts

```diff
@@ -0,0 +1,8 @@
+export async function lazyLoadProfiler() {
```

Comment:
Does this mean this only work in when the browser SDK is installed through npm?

Reply:
To my understanding, I think this lazy loading code will run whatever the installation process, but I can be wrong of course :) I'm no expert. From what I know, that's also how the Session Replay recorder is loaded.

Reply:
> To my understanding, I think this lazy loading code will run whatever the installation process

You're correct, this works with the CDN and with NPM given it's using a bundler.

Reply:
Perfect! Thank you!